evaluate_library_access_from_output_tsv/read_and_merge_library_access_datasets.R

```diff
@@ -0,0 +1,43 @@
+#### Read the datasets ####
+
+read_and_merge_library_access_datasets <- function(
+  lzma_compressed_library_access_tsv_location,
+  original_dataset_with_oa_color_column_location
+){
+  # '<<-' at several places below will add the variable into our global scope,
+  # beyond this function.
+  lzma_compressed_library_access_tsv <<- readr::read_tsv(
+    lzma_compressed_library_access_tsv_location
+  )
+  # View(lzma_compressed_library_access_tsv)  # Check the dataset
+  
+  # Create a temporary filepath for downloading the original dataset.
+  # Then download and read it.
+  tmp_filpath_for_original_dataset <- tempfile()
+  
+  download.file(
+    original_dataset_with_oa_color_column_location,
+    destfile = tmp_filpath_for_original_dataset,
+    mode = 'wb'
+  )
+  
+  original_dataset_with_oa_color_column <<- read.table(
+    gzfile(tmp_filpath_for_original_dataset),
+    sep = '\t',
+    header = TRUE
+  )
+  # View(original_dataset_with_oa_color_column)  # Check the dataset
+  
+  #### Merge the datasets ####
+  
+  # Combine the datasets so that we have doi, full_text_indicator,
+  # and oadoi_color
+  merged_datasets <- merge(
```

Comment:
Let's use `dplyr::inner_join` or `dplyr::left_join` (whichever join is appropriate).

Reply:
Implemented in f96571d3bbd00.